core/src/main/java/com/datastax/oss/driver/internal/core/session/DefaultSession.java

```diff
@@ -517,7 +548,6 @@ private void onChildrenClosed(List<CompletionStage<Void>> childrenCloseStages) {
                 if (!f.isSuccess()) {
                   closeFuture.completeExceptionally(f.cause());
                 } else {
-                  LOG.debug("[{}] Shutdown complete", logPrefix);
```

Comment:
why do you remove this log?

Reply:
It's replaced by the one on Line 372 (which covers more cases btw, since it's implemented as a generic callback on the close future).